src/test/java/shop/RealItemTest.java

```diff
@@ -0,0 +1,30 @@
+package shop;
+
+import org.junit.jupiter.api.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class RealItemTest {
+
+    @BeforeEach
+    void setUp() {
+        System.out.println("Starting the validation of the RealItem class");
+    }
+
+    @AfterEach
+    void tearDown() {
+        System.out.println("The validation of the RealItem class has been completed");
+    }
+
+    @Test
+    @DisplayName("Checking the setWeight() method")
+    void setWeight() {
```

Comment:
There is no needs to write unit tests for simple operations as get/set. There is no any logic in them, so that is redundant. RealItem and VirtualItem was designed as simple classes without any business logic. The only place, for which unit test can be written, is toString() method here. Please remove tests for set/get (it is optional to write tests for toString(), but you can do it)